eq-author/cypress/integration/authenticated/builder_spec.js

```diff
@@ -113,6 +113,27 @@ describe("builder", () => {
     );
   });
 
+  it("Can edit question additional information", () => {
+    const additionalInfoLabel = "this is some additionalInfo label";
+    cy.get(testId("txt-question-additional-info-label")).type(
+      additionalInfoLabel
+    );
+    cy.get(testId("txt-question-additional-info-label")).should(
+      "contain",
+      additionalInfoLabel
+    );
+
+    const additionalInfoContent = "this is some additionalInfo content";
+    typeIntoDraftEditor(
+      testId("txt-question-additional-info-content", "testid"),
+      additionalInfoContent
+    );
+    cy.get(testId("txt-question-additional-info-content", "testid")).should(
```

Comment:
I know we do this in a number of other places but, I've never checked if when the mutation fails this doesn't happen. I just want to know if all we are proving is that the RTE was updated which doesn't seem worthwhile as a cypress test.

Reply:
I think we do this all over the place so maybe we could raise a ticket for this and convert them to do that or remove them. Happy to leave it in this PR and deal with it later @samiwel